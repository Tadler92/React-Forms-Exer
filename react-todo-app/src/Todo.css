.Todo span {
  margin: 10px;
  font-size: 20px;
}

.Todo button {
  margin-bottom: 15px;
  background-color: crimson;
  color: white;
  padding: 5px 30px;
  border-radius: 30px;
}