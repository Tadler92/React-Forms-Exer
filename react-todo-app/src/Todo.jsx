import './Todo.css'

const Todo = ({taskTodo, removeTodo, id}) => {
  const remove = () => removeTodo(id);
  return (
    <div className="Todo">
      <span>{taskTodo}</span>
      <button onClick={remove}>X</button>
    </div>
  )
}


export default Todo;