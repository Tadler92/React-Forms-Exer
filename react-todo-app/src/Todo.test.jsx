import {render, fireEvent, waitFor} from "@testing-library/react";
import Todo from "./Todo";
import { expect, it } from "vitest";


describe("Todo Component", () => {
  it("renders without crashing", function () {
    render(<Todo />);
  });
  
  
  it("matches snapshot", function () {
    const {asFragment} = render(<Todo />);
    expect(asFragment()).toMatchSnapshot();
  });
});