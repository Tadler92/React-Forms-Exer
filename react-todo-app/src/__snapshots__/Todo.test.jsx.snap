// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Todo Component > matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
  >
    <span />
    <button>
      X
    </button>
  </div>
</DocumentFragment>
`;
